cli/build-cmd.ts

```
@@ -64,14 +64,22 @@
   return getEnvVersion(`refs/tags/v0.0.0-smoke-${gitHead}-${dateTick}`, xenv);
 }
 
-function patchDeps(dep: Record<string, string>, version: string) {
+function getPrefixedVersion(version: string, versionPrefix: string): string {
+  if (!versionPrefix) {
+    return version;
+  }
+  return `${versionPrefix}${version}`;
+}
+
+function patchDeps(dep: Record<string, string>, version: string, versionPrefix: string = "") {
```

ESLint

[error] Type string trivially inferred from a string literal, remove type annotation.

(@typescript-eslint/no-inferrable-types)